<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_personal</class>
 <widget class="QDialog" name="dialog_personal">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Personal</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>551</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>471</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Consultar</string>
     </attribute>
     <widget class="QTableView" name="tb_personal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>511</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CONSULTAR PERSONAL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_buscar">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>191</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buscar">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_nombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar por Nombre</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_apellido">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar por Apellido</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cedula">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar por Cedula</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Ingresar</string>
     </attribute>
     <widget class="QPushButton" name="btn_guardar">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>400</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>291</width>
        <height>328</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nombre</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Apellido</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Cedula</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Telefono</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Tipo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Direccion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Sexo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Correo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Carrera</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Facultad</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="txt_nombre">
           <property name="inputMethodHints">
            <set>Qt::ImhHiddenText</set>
           </property>
           <property name="maxLength">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_apellido">
           <property name="inputMethodHints">
            <set>Qt::ImhHiddenText</set>
           </property>
           <property name="maxLength">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_cedula">
           <property name="inputMethodHints">
            <set>Qt::ImhFormattedNumbersOnly</set>
           </property>
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_telefono">
           <property name="maxLength">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmb_tipo">
           <item>
            <property name="text">
             <string>Ayudante</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Personal Contratado</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Profesor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Voluntario</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_direccion">
           <property name="inputMethodHints">
            <set>Qt::ImhHiddenText</set>
           </property>
           <property name="maxLength">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmb_sexo">
           <item>
            <property name="text">
             <string>Masculino</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Femenino</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_correo">
           <property name="inputMethodHints">
            <set>Qt::ImhEmailCharactersOnly</set>
           </property>
           <property name="maxLength">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_carrera">
           <property name="inputMethodHints">
            <set>Qt::ImhHiddenText</set>
           </property>
           <property name="maxLength">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_facultad">
           <property name="inputMethodHints">
            <set>Qt::ImhHiddenText</set>
           </property>
           <property name="maxLength">
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>INGRESAR PERSONAL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_limpiar">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Eliminar</string>
     </attribute>
     <widget class="QPushButton" name="btn_eliminar_selec">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Seleccion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_eliminar_todo">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>370</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Personal</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ELIMINAR PERSONAL</string>
      </property>
     </widget>
     <widget class="QTableView" name="tb_personal_eliminar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>511</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
